core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -470,6 +471,15 @@ void setAttribute(AlluxioURI path, SetAttributeOptions options)
    */
   void scheduleAsyncPersistence(AlluxioURI path) throws AlluxioException;
 
+  /**
+   * Update the operation mode for the given ufs path under one or more mount points.
```

Comment:
When would a path be under multiple mount points? Isn't that disallowed at mount time?

For example, if I use ufs path "s3a://bucket/pathA/path2", does that mean a different path "s3a://bucket/pathB/" is unmodified?

Reply:
this should be "the given physical ufs path" w/ only scheme and authority. 

consider 2 mount points:
- `/mnt/mount1` -> `hdfs://ns1/folder1`
- `/mnt/mount2` -> `hdfs://ns1/folder2`

this method takes `hdfs://ns1` as arg and affects both mount points (persisted operations)